conf/postgres-operator/cluster-deployment.json

```diff
@@ -157,9 +157,6 @@
                         },
                         {{ end }}
                         {
-                            "mountPath": "/pgwal",
-                            "name": "pgwal-volume"
```

Comment:
This was present but never used. The container only touches `/pgwal` when `PGHA_WALDIR` env var is set.